x-pack/plugins/session_view/public/components/detail_panel_host_tab/index.tsx

```diff
@@ -5,194 +5,411 @@
+            title="Orchestrator"
+            listItems={[
+              {
+                title: <DetailPanelListItem>resource.ip</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`orchestrator.resource.ip: "${orchestratorData.resource.ip}"`}
+                    tooltipContent={orchestratorData.resource.ip}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {orchestratorData.resource.ip}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>resource.name</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`orchestrator.resource.name: "${orchestratorData.resource.name}"`}
+                    tooltipContent={orchestratorData.resource.name}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {orchestratorData.resource.name}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>resource.type</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`orchestrator.resource.type: "${orchestratorData.resource.type}"`}
+                    tooltipContent={orchestratorData.resource.type}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {orchestratorData.resource.type}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>namespace</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`orchestrator.namespace: "${orchestratorData.namespace}"`}
+                    tooltipContent={orchestratorData.namespace}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {orchestratorData.namespace}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>cluster.id</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`orchestrator.cluster.id: "${orchestratorData.cluster.id}"`}
+                    tooltipContent={orchestratorData.cluster.id}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {dataOrDash(processOrchestrator?.cluster?.id)}
```

Comment:
I think you can just use `orchestratorData` here

Reply:
woops i missed that